src/en/clouds.md

```diff
@@ -39,6 +39,22 @@ rackspace    rackspace   DFW, ORD, IAD, LON, SYD, HKG
 
 This lists the cloud name (which you will use to specify the cloud you want to 
 use), its type (the API used to control it) and the known regions for the cloud.
+The default region for a cloud is shown first in the list-clouds output, so
+in the above, `us-east-1` is the default region for an aws cloud. To specify
+a different region during controller creation, use:
+```bash
+juju bootstrap mycontroller aws/us-west-2
```

Comment:
We can mention region defaults (I didn't know this) but I think the actual example command should be in [controllers-creating](https://jujucharms.com/docs/devel/controllers-creating) and possibly provide a link to that page.